mob.go

```diff
@@ -1063,9 +1086,49 @@ func start(configuration Configuration) error {
 	sayInfo("you are on wip branch '" + currentWipBranch.String() + "' (base branch '" + currentBaseBranch.String() + "')")
 	sayLastCommitsList(currentBaseBranch.String(), currentWipBranch.String())
 
+	if configuration.OpenCommand != "" {
```

Comment:
I find these kind of checks scary. It means that we're handling this on a very low level, but everywhere. The invariant here is that it will always be either an empty string, or a specific string, but never anything in between. I would want to isolate this invariant in its own type. So that we abstract the low level detail away from the rest of the code. But that's just how I see it.

Reply:
What do you mean with a specific string here? The string can be any type of command which opens a file. For example you could set it to `idea %s` if you want to use IntelliJ, but it could also be `atom %s`, `code %s` or any other IDE or texteditor you can image. So I don't think we should specify how this open command can be set.

Reply:
I made some commits, tell me what you think. Now I wonder, if we need the `configuration.isOpenCommandGiven` at all, or we can just use `configuration.openCommandFor` which would be even better.